tests/benchmark/test_custream.custream.stdout

```diff
@@ -0,0 +1,11 @@
+ STREAM Benchmark implementation in CUDA
+ Array size (double precision) = 536.87 MB
+ using 1024 threads per block, 65536 blocks
+ output in IEC units (KiB = 1024 B)
+
+Function      Rate (GiB/s)  Avg time(s)  Min time(s)  Max time(s)
+-----------------------------------------------------------------
+Copy:         746.3174      0.00134171   0.00133991   0.00134277
+Scale:        745.7866      0.00134183   0.00134087   0.00134301
```

Comment:
You can change of the value to make sure the regex is able to parse an integer?
for instance: "746" instead of "746.3174".